examples/tomo/ray_trafo_parallel_2d_complex.py

```diff
@@ -0,0 +1,42 @@
+"""Example computing the 2d parallel beam ray transform of a complex phantom.
+
+This example demonstrates that complex-valued functions work for both
+forward and back projections by application to real and imaginary parts
+separately.
+"""
+
+import numpy as np
+import odl
+
+# Reconstruction space: discretized complex-valued functions on the rectangle
+# [-20, 20]^2 with 300 samples per dimension.
+reco_space = odl.uniform_discr(
+    min_pt=[-20, -20], max_pt=[20, 20], shape=[300, 300], dtype='complex128')
```

Comment:
Actually I wanted `'complex64'` to match the real space example, this was only to test if other dtypes work, too.